### DISCUSSION TITLE
How to fix [vite-node] Failed to load module?

### DISCUSSION THREAD

**Discussion by PavelDemyanenko** [vitest-dev/vitest#1385] [Q&A] [answered]
Hi, upon testing react components i get this error message: Error: [vite-node] Failed to load 'module name'. What is the issue here and how to fix it?

**Comment by sheremet-va** [comment]
Can be anything. You might misspelled it. You might not have it installed. The path might be wrong.

**Reply by PavelDemyanenko** [reply]
Those are tests which successfully ran with jest, but after i've added vitest and tried to run with it, it fails.

**Comment by PavelDemyanenko** [comment, marked as answer]
the issue is that i used absolute path for module importa from root dir, and vitest didn't understand this. i had to rename to relative paths in app.

**Comment by AdrianTNare** [comment]
in my case i had a path like  ` src/folder/folder2`  and  vitest wont know the actual location of src, so in this case we should use an alias
to help vite resolve src . For example by adding 
```
resolve: {
    alias: {
      src: path.resolve(__dirname, "./src"),
    },
  },
```
to our vitest.config.js
